openpype/hosts/traypublisher/plugins/publish/collect_simple_instances.py

```diff
@@ -140,128 +123,68 @@ def _create_main_representations(
                 os.path.join(filepath_item["directory"], filename)
                 for filename in filepath_item["filenames"]
             }
+
+            # add it to source filepaths
+            # for later validation of existence
             source_filepaths.extend(filepaths)
 
-            source = self._calculate_source(filepaths)
-            representation = self._create_representation_data(
-                filepath_item, repre_names_counter, repre_names
-            )
-            instance.data["representations"].append(representation)
-            representation_files_mapping.append(
-                (filepaths, representation, source)
-            )
-        return source
-
-    def _create_review_representation(
-        self,
-        instance,
-        source_filepaths,
-        repre_names_counter,
-        repre_names,
-        representation_files_mapping
-    ):
-        # Skip review representation creation if there are no representations
-        #   created for "main" part
-        #   - review representation must not be created in that case so
-        #       validation can care about it
-        if not representation_files_mapping:
-            self.log.warning((
-                "There are missing source representations."
-                " Creation of review representation was skipped."
-            ))
-            return
+            file_path = self.get_single_filepath_from_list_of_files(
+                filepaths)
+            frame_start, frame_end = self.get_frame_range_from_list_of_files(
+                filepaths)
+            processing_file_data = {
+                file_path: {
+                    "framerange": (frame_start, frame_end)
```

Comment:
As far as I can tell, the method returns a `tuple` and we are not checking if they are non, so we can skip line 133 and call it here directly:
```py

processing_file_data = {
	file_path: {
		"framerange": self.get_frame_range_from_list_of_files(
			filepaths
		)
	}
}
```